### ISSUE TITLE
CJK cannot display for native Linux client games

### ISSUE THREAD

**Issue by NomadThanatos** [ValveSoftware/SteamOS#1210]
#### Your system information

* Steam client version: 1700160213
* SteamOS version: 3.5.5
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
I had this issue for RimWorld & Gremlins, Inc.
And using Proton to play above games will resolve this.

![image](https://github.com/ValveSoftware/SteamOS/assets/5603905/674be461-b3ab-4619-a586-73c71a7b85a9)
Above should have Chinese characters in the menu.

![image](https://github.com/ValveSoftware/SteamOS/assets/5603905/54ca7ecb-3f89-4206-a6fa-e27d198a7b8f)
Above should have Chinese, Japanese, and Korean characters in the bracket as the rest languages.

I had this [issue](https://steamcommunity.com/app/1675200/discussions/1/3727324132814445071/) long time ago and heard Chinese is supported in 3.5.5 for desktop mode, thought this might be fixed, but after my tests, both games are still having this issue.

Those screenshots are taken from that time, but it shows the same with 3.5.5

#### Steps for reproducing this issue:

1. Install RimWorld or Gremlins, Inc
2. Run in with native Linux client (without Proton)
3. Change the language to Chinese, Japanese, or Korean.

**Comment by AG-w** [comment]
it's indeed fixed, you need to add locale
https://github.com/ValveSoftware/SteamOS/issues/794

**Comment by NomadThanatos** [comment]
Hi, I tried both games with
`LC_ALL=zh_CN.UTF-8 %command%`
And still have this issue, would be appreciating for any instructions about how to set it up.

**Comment by AG-w** [comment]
you edit `/etc/locale.gen` then generate new locale
https://wiki.archlinux.org/title/locale#Generating_locales

**Comment by bertogg** [comment]
Hi, a couple of things:

- SteamOS 3.5 comes with `zh_CN.utf8` already there's no need to generate anything.
- These are anyway Steam games, right? Maybe it's a bug in the Linux versions of those games, but I'll try to have a look.

**Comment by NomadThanatos** [comment]
Yeah those are Steam games, it could be a bug for those games, in that case I will create tickets for the developers.

But since there are not much Steam Linux games and I have seen these 2 games with exactly the same issue, let's confirm it is not a bug with SteamOS.

Thank you!

**Comment by bertogg** [comment]
There are definitely native Linux games that do CJK just fine, I just tested Portal 2 for example.

**Comment by nisehime** [comment]
It's a Unity issue
https://github.com/ValveSoftware/steam-runtime/issues/555
